src/__test__/e2e/signing/evm-abi.test.ts

```diff
@@ -158,6 +172,7 @@ describe('[EVM ABI]', () => {
           const idx = (i * setSz) + j;
           if (idx < maxVec) {
             it(`(Canonical #${idx}/${maxVec}) ${vectors.canonicalNames[idx]}`, async () => {
+              encDefsCalldata[idx] = '0x7a8555b900000000000000000000000038a1e0bf2745740c303fe4140397d157818a6e3c000000000000000000000000000000000000000000000000000000000000000e0000000000000000000000000000000000000000000000000000000000000026000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000004d0200000000000000000000000000000000000000000000000000000000000000410000000000000000000000000000000000000000000000000000000000003a95000000000000000000000000000000000000000000000000000000000000014000000000000000000000000000000000000000000000000000000000000175800000000000000000000000000000000000000000000000000000000000000041254e600d62d2023c81f092c2f95951cc1e540c572230eb00d57e2ee613e44fb34ca84461f23fc8be1eea17a4c3c95c6c08fe1ba2c9cb55de326919947c9b947b1b00000000000000000000000000000000000000000000000000000000000000'
```

Comment:
Oops this is cruft

Reply:
Removed [ce4df1a](https://github.com/GridPlus/gridplus-sdk/pull/483/commits/ce4df1a62b2472b0ee54ab5317d4c90439cf9aa1)